Automattic/wp-calypso
# Add step for Jetpack Cancellation flow to show plan/ product information and benefits

#### Changes proposed in this Pull Request

* This PR is related to/ a subpart of https://github.com/Automattic/wp-calypso/pull/54669 which introduces a new cancellation flow for Jetpack products and plans. 
* This PR adds the Benefits step component for the cancellation flow 

#### Testing instructions

The visual output of this component can be seen by testing PR #54669
* Either check out the branch from PR #54669 locally or use the Calypso live link on that branch to get a preview. 
* Review the testing instructions on PR #54669 to check the visual output
